__tests__/app.test.js

```diff
@@ -169,6 +164,63 @@ test("Testing for a 404 error with a invalid review_id that does not exist", ()
     });
 });
 
+test("GET /api/reviews?query accepts a sort_by query, which sorts by title", () => {
+  return request(app)
+    .get("/api/reviews?sort_by=title")
+    .expect(200)
+    .then(({ body: { reviews } }) => {
+      expect(reviews).toBeSortedBy("title", { descending: true });
+    });
+});
+test("accepts query and returns the reviews in default order (date and descending)", () => {
+  return request(app)
+    .get(`/api/reviews`)
+    .expect(200)
+    .then(({ body: { reviews } }) => {
+      expect(reviews).toBeSortedBy("created_at", { descending: true });
+    });
+});
+test("accept a category and return games in that category", () => {
+  return request(app)
+    .get(`/api/reviews?category=social+deduction`)
+    .expect(200)
+    .then(({ body: { reviews } }) => {
+      expect(reviews).toBeSortedBy("title", { descending: false });
+    });
+});
+test("return reviews ordered by title in ascending order", () => {
+  return request(app)
+    .get(`/api/reviews?sort_by=designer&&order=asc`)
+    .expect(200)
+    .then(({ body: { reviews } }) => {
+      expect(reviews).toBeSortedBy("designer", { descending: false });
+    });
+});
+test("testing for a status 400, user enters non-valid sort_by query", () => {
+  return request(app)
+    .get("/api/reviews?sort_by=badrequest")
+    .expect(400)
+    .then(({ body }) => {
+      expect(body.msg).toBe("Bad request");
+    });
+});
+test("testing for 400 status when order query is invalid", () => {
+  return request(app)
+    .get(`/api/reviews?sort_by=title&&order=phone`)
+    .expect(400)
+    .then(({ body }) => {
+      expect(body.msg).toBe("Bad request");
+    });
+});
+test("testing for a 404 when a category column is invalid", () => {
+  return request(app)
+    .get(`/api/reviews?category=nothing`)
+    .expect(404)
+    .then(({ body }) => {
+      expect(body.msg).toBe("Category not found");
+    });
+});
+
```

Comment:
Add another test when the category is valid, but there is no reviews in that category (have a look in the db, there is definitely one!). It should ideally return an empty array.